test_rclcpp/CMakeLists.txt

```diff
@@ -104,6 +104,18 @@ if(BUILD_TESTING)
   # Macro for tests that trigger the shutdown of an executable based on particular console output,
   # then check the output of the executable against different console output.
   macro(custom_launch_test_executable_output test_name executable)
+    cmake_parse_arguments(_ARG "SKIP_TEST" "TIMEOUT" "" ${ARGN})
+    if(_ARG_UNPARSED_ARGUMENTS)
+      message(FATAL_ERROR "custom_launch_test_executable_output() called with "
+      "unused arguments: ${_ARG_UNPARSED_ARGUMENTS}")
+    endif()
+    if(_ARG_TIMEOUT)
+      set(_ARG_TIMEOUT "TIMEOUT" "${_ARG_TIMEOUT}")
+    endif()
+    if(_ARG_SKIP_TEST)
+      set(_ARG_SKIP_TEST "SKIP_TEST")
+    endif()
+
```

Comment:
you're right, I thought it was necessary but it was only because I tried to pass ARGN as the last argument to `ament_add_nose_test`. Removed in https://github.com/ros2/system_tests/pull/215/commits/186356c19bc3345b9a8d28c3e73b5a78a1115dc7